### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by elias1shock** [MicroTrendsLtd/NinjaTrader8#100] [Q&A]
Hi, my name is Elias Shokry. I have a degree in data analysis from Georgia Tech, and have 3 years of experience in Electronic Trading. I would love to help you with this project but unfortunately I am not familiar with ninja trader. I have been able to bypass a few compilation errors but cannot get by this one. Could you help me?

Error Message: 
The type 'NinjaTrader.NinjaScript.Strategies.STrategy' already contains a definition for 'indicator'
Type 'NinjaTrader.NinjaScriptStrategies.Strategy' already defines a member called 'Strategy' with the same parameter types

**Comment by MicroTrendsTom** [comment]
sounds like duplicate folder stucture?
